packages/react-components/lib/reports/multi-level-menu.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
```

Comment:
Can you add a story for this?

Reply:
Added!

Reply:
![image](https://user-images.githubusercontent.com/11761240/112028134-9e852780-8b16-11eb-9aee-69312eb6fd82.png)